### ISSUE TITLE
Make LRCLIB as seperate source option along with musixmatch and YouTube transcript 

### ISSUE THREAD

**Issue by Its-Tony** [maxrave-dev/SimpMusic#419]
Musixmatch doesn't show some lyrics.Spotify Api isn't working anymore( return 5 unsyced lines only) and youtube transcript work only with Videos. So Try to make three option inside main lyrics provider option(include LRCLIB).sometime it shows lyrics page without any word. In current debug I think you have priorities musixmatch,if musixmatch fail to return data then it try with Lrclib. Please make LRCLIB as seperate option for Lyrics. Have a look at #415 and now see the real problem why it is fail to load lyrics for some song. Musixmatch shows "not authorised to show lyrics for this track".
![Screenshot_2024-06-29-19-02-49-669_com android chrome](https://github.com/maxrave-dev/SimpMusic/assets/172184183/ca73c2e4-1f9b-4c12-88d8-7df843573e7b)

**Comment by assier08** [comment]
Yeah, in my case I only have the lyrics of a few songs, and also musixmaxh doesn't let me log in without using a Google account
